### ISSUE TITLE
plo1 TEV site

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3558]
@manulera  I wondered if you could help me with this one (top row and bottom row)

We have 2 TEV AA341
I could alter one, but I wondered if we should represent these as  partial deletions in PomBase?

eiter way, one seems to have 2 TEV sites, how should this be names  (and does it need the plo1 twice?)
<img width="945" alt="Screenshot 2023-09-11 at 09 09 25" src="https://github.com/pombase/curation/assets/7359272/4e4c535b-812f-464d-91cd-a8d7b8e6623c">

**Comment by ValWood** [comment]
Re 
warning in 2a8be2bb0239204b: description for new allele "plo1-TEVAA341(tev-site@AA341)" does not match the existing allele with the same name "plo1-TEVAA341(plo1(1-341)-(ENLYFQGAS)-plo1(342-405)-(ENLYFQGAS)-plo1(406-683))" (from session 2a8be2bb0239204b)

**Comment by manulera** [comment]
I wrote this description `plo1(1-341)-(ENLYFQGAS)-plo1(342-405)-(ENLYFQGAS)-plo1(406-683)`. That's the  way of describing the allele with the syntax we have proposed for chimeras/fusions. It should include plo1 more than once, because it could be a chimera of plo1 and another protein using that same syntax.

Also, is there any annoation to the allele named differently? If not, I would just delete it.
